src/component.ts

```diff
@@ -10,6 +10,15 @@ export class Component extends BaseSDK {
 		super({});
 		this._id = props.componentId;
 		this.type = "Component";
+		if (props?.manifestMethods?.length > 0) {
```

Comment:
first optional chain check is not needed

Reply:
removed